src/files/domain/useCases/GetDatasetFilesTotalDownloadSize.ts

```diff
@@ -14,11 +15,15 @@ export class GetDatasetFilesTotalDownloadSize implements UseCase<number> {
     datasetId: number | string,
     datasetVersionId: string | DatasetNotNumberedVersion = DatasetNotNumberedVersion.LATEST,
     fileDownloadSizeMode: FileDownloadSizeMode = FileDownloadSizeMode.ALL,
+    fileSearchCriteria?: FileSearchCriteria,
+    includeDeaccessioned: boolean = false,
   ): Promise<number> {
     return await this.filesRepository.getDatasetFilesTotalDownloadSize(
       datasetId,
       datasetVersionId,
+      includeDeaccessioned,
       fileDownloadSizeMode,
+      fileSearchCriteria,
```

Comment:
Why is the order of the arguments different in the use case than in the filesRepository? 

it's not that important but if you choose one order why not staying with it?

Reply:
The order is different because in the repository method the _includeDeaccessioned_ parameter no longer has a default value, since this is determined by the use case.

I cannot put _includeDeaccessioned_ as the last parameter in the repository method since TypeScript does not allow putting a non-optional parameter followed by an optional one (_fileSearchCriteria_ is optional)

I want to leave includeDeaccessioned as the last parameter in the use case so that we don't have to specify it when calling the use case if not necessary.